include/boost/spirit/home/qi/detail/expectation_failure.hpp

```diff
@@ -0,0 +1,31 @@
+/*=============================================================================
+Copyright (c) 2001-2011 Joel de Guzman
+Copyright (c) 2016      Frank Hein, maxence business consulting gmbh
+
```

Comment:
Not a biggie, but you probably shouldn't claim copyright on a file with contents lifted elsewhere. The basic rule here is that you add your copyright if there's significant contribution to the code in terms or actual content.